src/flavors/central-puget-sound/config.json

```diff
@@ -204,22 +204,13 @@
         "id": "mwfeatured",
```

Comment:
Note that many changes below in config files are just the result of saving the file for the first time with Prettier configured. The relevant config section for this PR is called `pages`.